packages/next/src/client/components/router-reducer/fill-lazy-items-till-leaf-with-head.ts

```diff
@@ -11,7 +11,7 @@ export function fillLazyItemsTillLeafWithHead(
   routerState: FlightRouterState,
   cacheNodeSeedData: CacheNodeSeedData | null,
   head: React.ReactNode,
-  wasPrefetched?: boolean
+  hasReusablePrefetch?: boolean
```

Comment:
I think is argument is no longer needed? For the same reason you removed it from the PPR implementation

Reply:
I think it's still needed but for a more subtle reason -- we set this flag to `true` specifically for `auto` prefetches that don't have a `reusable` cache status. So that means fresh entries technically don't fall into this categorization. 

When attempting to remove it I was noticing that once we renewed the stale prefetch, the lazy fetch was never happening, because `fillLazyItemsTillLeafWithHead` would always hit [this condition](https://github.com/vercel/next.js/blob/f507d83c36af1173d66b6c83b46a8b95405dab6d/packages/next/src/client/components/router-reducer/fill-lazy-items-till-leaf-with-head.ts#L62) since there would always be an existing cache node. 

For now I'm going to leave it as-is but I'd like to better understand why `fillLazyItemsTillLeafWithHead` is triggering any lazy fetch behavior rather than `fillCacheWithNewSubtreeData`.